src/Marten/IAdvancedSql.cs

```diff
@@ -62,6 +114,7 @@ public interface IAdvancedSql
     /// <param name="sql"></param>
     /// <param name="parameters"></param>
     /// <returns></returns>
+    [Obsolete(QuerySession.SynchronousRemoval)]
     IReadOnlyList<T> Query<T>(string sql, params object[] parameters);
```

Comment:
A couple of spots where the obsolete warning for synchronous db methods was missing.